frontend/packages/noobaa-storage-plugin/src/components/data-consumption-card/data-consumption-card.tsx

```diff
@@ -0,0 +1,300 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import {
+  Chart,
+  ChartAxis,
+  ChartBar,
+  ChartGroup,
+  ChartLegend,
+  ChartThemeColor,
+} from '@patternfly/react-charts';
+import { ChartAreaIcon } from '@patternfly/react-icons';
+import { EmptyState, EmptyStateVariant, EmptyStateIcon, Title } from '@patternfly/react-core';
+
+import {
+  DashboardCard,
+  DashboardCardBody,
+  DashboardCardHeader,
+  DashboardCardTitle,
+} from '@console/internal/components/dashboard/dashboard-card';
+import {
+  DashboardItemProps,
+  withDashboardResources,
+} from '@console/internal/components/dashboards-page/with-dashboard-resources';
+import { humanizeDecimalBytes } from '@console/internal/components/utils';
+import { PrometheusResponse } from '@console/internal/components/graphs';
+
+import {
+  ACCOUNTS,
+  EGRESS,
+  IOPS,
+  LOGICAL_USAGE,
+  PHYSICAL_VS_LOGICAL_USAGE,
+  PROVIDERS,
+} from './constants';
+import {
+  accountsLogicalUsageChartData,
+  accountsLogicalUsageChartLegendData,
+  iopsChartData,
+  iopsChartLegendData,
+  providersEgressChartData,
+  providersEgressChartLegendData,
+  providersPhysicalVsLogicalChartData,
+  providersPhysicalVsLogicalChartLegendData,
+} from './data-consumption-card-utils';
+import { DataConsumptionBody } from './data-consumption-card-body';
+import { DataConsumptionDropdown } from './data-consumption-card-dropdown';
+import { DataConsumptionQueries } from '../../queries';
+import { getMetric, getValue } from '../../utils';
+import './data-consumption-card.scss';
```

Comment:
Rest of the code has just single scss import per plugin/application which subsequently `@import` specialized fragments.
I think we should stay consistent.

IMO, this per-component encapsulation fits better a library than an end-user application.

@spadgett , thoughts?

Reply:
@christianvogt @vojtechszocs opinion?

Reply:
My thought is that SCSS is no different than a js module. If a class is specific to a component and no other component, then why not import it at the component level where it belongs. This will also benefit from lazy loading. If we need to refactor / remove the component. Finding the a related CSS code is also easier. It ensures we are being explicit in the dependencies of that component.

We don't use utilities from the global scope. CSS should be no different. I'd even argue that utility classes should also be imported at the component level as well and let webpack chunk those utility classes into a common module at the root. This happens for modules like lodash which is used everywhere.

I'm 100% for co-locating CSS and trying to make as scoped as possible. If we could eventually do what PF4 does, even better.

The `dev-console` follows this pattern.